content/blog/convince-my-boss.md

```diff
@@ -0,0 +1,21 @@
+Dear \{bossName\},
+
+I am excited to brief you about PyCon India, India’s premier conference on using and developing the Python programming language that takes place in Chennai from October 12\-15, 2019\. About a 1000\+ of the industry's Pythonistas from different fields are bound to be in attendance for the 11th edition of this mega event\.
+
+It would be great if I get an opportunity to attend this Conference as there will be Innumerable benefits for me as an individual which will in turn impact the Organization as well\. To list a few:
+
+- Am Particularly keen on attending this <talk/workshop/devsprint> on <topics> presented by <speaker name> as our project is based on this <or upcoming project based on this>
```

Comment:
I am not sure how this would work. Talks and workshops are not confirmed till 2 months before the conference. But the tickets open now. People will not know the topics and speakers, except for the keynote speaker.